kayenta-objectstore-configbin/src/main/java/com/netflix/kayenta/configbin/storage/ConfigBinStorageService.java

```diff
@@ -29,7 +29,7 @@
 import com.netflix.kayenta.security.AccountCredentialsRepository;
 import com.netflix.kayenta.storage.ObjectType;
 import com.netflix.kayenta.storage.StorageService;
-import com.netflix.spinnaker.kork.core.RetrySupport;
+import com.netflix.kayenta.util.Retry;
```

Comment:
It occurred to me after I finished reviewing the PR that I had seen this logic before. I search kayenta on github and ended up back at this file. Wouldn't it make more sense to just add the new logic to the existing `RetrySupport` class in kork?

Reply:
This is a temporary step along the path to make things more context aware, so I'd prefer to keep it inside kayenta for now.